file-formats/aifi/type/zif_aff_aifi_v1.intf.abap

```diff
@@ -0,0 +1,506 @@
+      "! $default '/AIF/STD_IDX_TBL'
+      " (type /aif/msg_tbl)
+      index_table              TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Interface Display Name</p>
+      "! Interface display name
+      " (type /aif/ifdisplay)
+      display_name             TYPE c LENGTH 120,
+      "! <p class="shorttext">Application Log Lifetime</p>
+      "! Application log lifetime
+      "! $default '90'
+      " (type /aif/msg_lifetime)
+      app_log_lifetime         TYPE n LENGTH 10,
+      "! <p class="shorttext">Direction</p>
+      "! Direction
+      " (type /aif/ifdirection)
+      direction                TYPE ty_direction_type,
+      "! <p class="shorttext">Preprocessing</p>
+      "! Preprocessing
+      " /AIF/PRE_PROCESSING
+      preprocessing            TYPE abap_bool,
+      "! <p class="shorttext">Proxy XML Transformation</p>
+      "! Use proxy xml transformation
+      " /AIF/PROXY_XML_TRANSFORMATION
+      proxy_xml_transformation TYPE abap_bool,
+      "! <p class="shorttext">Extend XML Handling</p>
+      "! Extend xml handling
+      " /AIF/PRX_XXML
+      extend_xml_handling      TYPE abap_bool,
+      "! <p class="shorttext">Postprocessing</p>
+      "! Postprocessing
+      postprocesssing          TYPE abap_bool,
+    END OF ty_aif_interface.
+
+  TYPES:
+    "! <p class="shorttext">Recipient</p>
+    "! Recipient
+    BEGIN OF ty_recipient,
+      "! <p class="shorttext">Alert Recipient</p>
+      "! Alert recipient
+      "! $required
+      " (type /aif/recipient_object_name)
+      recipient TYPE c LENGTH 40,
+    END OF ty_recipient.
+
+  "! <p class="shorttext">Recipients</p>
+  "! Recipients
+  TYPES ty_recipients TYPE STANDARD TABLE OF ty_recipient WITH DEFAULT KEY.
+
+  TYPES:
+    "! <p class="shorttext">General</p>
+    "! General key field settings
+    BEGIN OF ty_keyfield_settings,
+      "! <p class="shorttext">Key Field Name</p>
+      "! Key field name
+      "! $required
+      " (type /aif/key_fieldname, field of index table)
+      key_field_name    TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Data Element</p>
+      "! Data element
+      "! $required
+      " (type rollname)
+      data_element      TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Name of Select-Option/Parameter</p>
+      "! Name of select-option/parameter
+      " (type /aif/stexti)
+      selopt_name       TYPE c LENGTH 8,
+      "! <p class="shorttext">Field Name</p>
+      "! Field name in define key fields
+      " (type /aif/lfieldname_kflds, field of sap/raw structure)
+      field_name        TYPE c LENGTH 120,
+      "! <p class="shorttext">SAP or RAW Structure</p>
+      "! Is sap or raw structure?
+      " (type /aif/raw_or_sap_kflds)
+      sap_raw_structure TYPE ty_sap_raw_structure_type,
+      "! <p class="shorttext">Select-Option</p>
+      "! Is select-option?
+      " /AIF/FLD_IS_SELECT_OPT
+      is_selopt         TYPE abap_bool,
```

Comment:
```suggestion
      is_select_option         TYPE abap_bool,
```

If you use the `is_` prefix here, maybe it would be better if all booleans are consistent with `is_`?

Reply:
is_ and uses_ are now in place